src/main/java/seedu/address/model/recipe/Name.java

```diff
@@ -3,13 +3,15 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
+import seedu.address.commons.util.StringUtil;
+
 /**
  * Represents a Recipe's name in the recipe book.
  * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
  */
 public class Name {
 
-    public static final String MESSAGE_CONSTRAINTS = "Names should not be left blank";
+    public static final String MESSAGE_CONSTRAINTS = "Names should contain valid alpha-numeric characters.";
```

Comment:
What was the reason why we can accept symbols again?

Reply:
No idea actually LOL we only checked for blank names initially, wanted to allow "&" also but one of the test case recipe name was "James&" so decided to just stick to alpha-numeric only

Reply:
Should recipe names be allowed to have those though? Shouldn't something like "Grandma's Special Fish & Chips" be valid?

Reply:
Potential issues from allowing special characters is names like "grandma recipe'&'" is allowed, which could be smth we could allow??? need discuss i think